recipes/python-codon-tables/LICENSE

```diff
@@ -0,0 +1,31 @@
+Copyright Status
```

Comment:
I thought there was no license, where does this one come from?

Reply:
I added that license as a guess, to ensure that the rest of the build completes without error. Once a decision on the license is reached, will update the license file here.

Reply:
Ahh ok. In that case it's better to not include a file as otherwise it might get accidentally merged.